srcpkgs/roccat-tools/INSTALL.msg

```diff
@@ -0,0 +1,2 @@
+Add users to the "roccat" group to allow them to use the tools to configure
```

Comment:
don't think this is necessary

Reply:
Without adding users to the group, the program just gives a "Permission denied" error. I'd rather leave the message in, at least if I change the group to `_roccat`. Other packages also use such a message (e.g. wireshark and weston)